app/views/people/membership_invoices/new.html.haml

```diff
@@ -0,0 +1,35 @@
+- if invoice_possible?(@member, @date)
+  .alert.alert-info=t(".alert_info")
+  
+  = standard_form(@invoice_form, url: group_person_membership_invoices_path(@group, @person), method: :post) do |f|
+    %h1 Mitgliedschaftsrechnung erstellen
+    = f.labeled(:reference_date) do
+      = f.date_field(:reference_date, class: 'date col-6', minDate: Date.today.beginning_of_year, maxDate: (Date.today.end_of_year + 1.year))
+    = f.labeled(:invoice_date) do
+      = f.date_field(:invoice_date, class: 'date col-6', minDate: Date.today.beginning_of_year, maxDate: (Date.today.end_of_year + 1.year))
+    = f.labeled(:send_date) do
+      = f.date_field(:send_date, class: 'date col-6', minDate: Date.today.beginning_of_year, maxDate: (already_member_next_year?(@person) ? Date.today.end_of_year + 1.year : Date.today.end_of_year))
+
+    = f.labeled(:section_id) do
+      - main_section = @person.sac_membership.stammsektion_role.layer_group
+      = f.inline_radio_button :section_id, main_section.id, t(".mv_yearly_invoice"), true, checked: true
+
+      .nested-radio-group
+        = f.labeled(:new_entry, class: "ms-3") do
+          = f.inline_radio_button :new_entry, true, t("global.yes"), true
+          = f.inline_radio_button :new_entry, false, t("global.no"), true, checked: true
+      
+      - currently_paying_zusatzsektionen(@member).each do |section|
+        = f.inline_radio_button :section_id, section.id, "#{t(".zusatzsektion_eintrittsrechnung")} #{section.name}", true, checked: false
+
+    = f.labeled(:discount) do
+      = f.inline_radio_button :discount, 0, t("global.no"), true, checked: true
+      = f.inline_radio_button :discount, 50, "50%", true
+      = f.inline_radio_button :discount, 100, "100%", true
+
+    = f.indented do
+      = submit_button(f, "Rechnung erstellen")
```

Comment:
muss in die locales oder kann vielleicht via activemodel name von der invoice_form gemacht werden